
syntax = "proto3";

package backend;

option go_package = "github.com/priyanshujain/infragpt/services/backend/backendapi/proto";

service BackendService {
  rpc SendReply(SendReplyCommand) returns (Status);
}

message SendReplyCommand {
  string conversation_id = 1;
  string message = 2;
}

message Status {
  bool success = 1;
  string error = 2;
}